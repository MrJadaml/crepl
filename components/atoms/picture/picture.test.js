import React from 'react';
import { Picture } from './index';
import { cleanup, render } from '@testing-library/react';
import { getByText } from '@testing-library/dom';

afterEach(cleanup);

describe('Picture', () => {
  it('should render an anchor tag', () => {
    const { asFragment } = render(
      <Picture
        media="(min-width: 800px)"
        src="/media/examples/painted-hand-298-332.jpg"
        srcset="/media/examples/surfer-240-200.jpg"
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
