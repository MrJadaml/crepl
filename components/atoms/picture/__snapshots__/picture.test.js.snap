// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Picture should render an anchor tag 1`] = `
<DocumentFragment>
  <picture
    class="picture"
  >
    <img
      class="image"
      src="/media/examples/painted-hand-298-332.jpg"
    />
  </picture>
</DocumentFragment>
`;
